example/example_resourcemanager_test.go

```diff
@@ -0,0 +1,35 @@
+package example
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/oracle/oci-go-sdk/common"
+	"github.com/oracle/oci-go-sdk/example/helpers"
+	"github.com/oracle/oci-go-sdk/resourcemanager"
+)
+
+func ExampleResourceManagerListStacks() {
+	client, err := resourcemanager.NewResourceManagerClientWithConfigurationProvider(common.DefaultConfigProvider())
+	helpers.FatalIfError(err)
+
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
how do you determine the time out values here? Other customer might copy and past samples to their code, any specific reason to add it here?

Reply:
Just come from my own mind,  I will remove it later on.